### ISSUE TITLE
apical dyskinesis

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#10627]
**HPO term**
Reduced contraction of the left ventricular apex HP:0031483

**Synonym to be added**
apical dyskinesis
apical hypokinesis

**broad/exact/narrow/plural/abbreviation ?**

**Comment by pnrobinson** [comment]
@carolina-f 
I do not think the above are synonyms.

Dyskinesis refers to an abnormal, often paradoxical, movement of the heart muscle. In the context of the left ventricular apex, this means that during systole (the phase of the heartbeat when the heart contracts to pump blood), the apical region moves in an abnormal direction. Instead of contracting and moving inward, it may bulge outward.

Hypokineses is a synonym of the existing term.

Thoughts?

**Comment by carolina-f** [comment]
The term hypokinesis is not currently listed as a synonym for Reduced contraction of the left ventricular apex HP:0031483.


I made the suggestion of the two terms based on the following publications about Takotsubo cardiomyopathy:

 PMID: 28613549: The most widely accepted criteria are the Mayo Clinic diagnostic criteria for the identification of stress cardiomyopathy. Outlined below are the key features; all are required to meet the diagnosis:
Transient hypokinesis, akinesis, or dyskinesis in the left ventricular mid segments with or without apical involvement; regional wall motion abnormalities that extend beyond.

PMID: 36216626 We describe a case of a six-month-old male diagnosed with Takotsubo cardiomyopathy in the setting of an accidental clonidine overdose. The patient presented with altered mental status and hypertension. In the course of his broad workup, cardiac dysfunction was indicated by bedside ultrasound in the Emergency Department. The classic apical dyskinesis was seen on a follow-up, cardiology-based echocardiogram. 

PMID: 30407769 Case: A 75-year-old patient with a family history of TCM was admitted with suspected ST-segment elevation myocardial infarction. Transthoracic echocardiography showed apical dyskinesis with hyperdynamic basal walls and a left ventricular ejection fraction (LVEF) of 25%. Repeat echocardiography showed normal LVEF of 60% ejection fraction. Cardiac catheterization showed no significant stenosis.

**Comment by pnrobinson** [comment]
@carolina-f 
Adding new term: 
- ID: HP:6001073
- label: Left ventricular apical dyskinesis
- parents: [Abnormal regional left ventricular contraction (HP:0031482)](https://hpo.jax.org/app/browse/term/HP:0031482)
- pmid: PMID:37184762
- def: Dyskinesis is used to describe ventricular muscle segments that bulge outside the left ventricular cavity during systole, as opposed to contracting inwards with the rest of the segments.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10627
- synonyms: 


I will also add "hypokinesis" to the existing term as a synonym
